sdk/storage/azure-storage-blob/tests/get_blob_to_stream_async.temp.b349881e-0f45-4eb2-aeb3-296aa3649006.dat

```diff
@@ -1,138 +1 @@
-interactions:
```

Comment:
Can you remove these `dat` files? These are temp files some of the tests and usually clean up but if a test fails, they don't always get cleaned up. They should not be checked in though.

Reply:
Will do, did not even see these!